th2_data_services_lwdp/commands/http.py

```diff
@@ -888,11 +898,11 @@ def __init__(
         self._decode_error_handler = decode_error_handler
         self._cache = cache
         self._page = page
-        self._start_timestamp = page.start_timestamp["epochSecond"]
+        self._start_timestamp = _check_millisecond(_seconds2ms(page.start_timestamp["epochSecond"]))
```

Comment:
Why we take `epochSecond` only but not `nano` ?
{'epochSecond': 1668003240, 'nano': 0},

Reply:
Don't create your own function for it, we already have one

Reply:
> Why we take `epochSecond` only but not `nano` ? {'epochSecond': 1668003240, 'nano': 0},

The method which converts datetime to seconds only takes int part of given timestamp nanoseconds become irrevelant

Reply:
> Don't create your own function for it, we already have one

Where can I find it?

Reply:
We have it in the DS lib 1.x 
https://github.com/th2-net/th2-data-services/blob/master/th2_data_services/provider/utils/converters.py
Let's copy it to DS core 2.0 but with name ProtobufTimestampConverter

Reply:
> The method which converts datetime to seconds only takes int part of given timestamp, so nanoseconds become irrevelant

And so?
We shouldn't consider milliseconds in `nanos` ?

Reply:
Now methods take
```
milliseconds  = nanosecond[:-6] # First 3 Digits
timestamp = [seconds][milliseconds] # Int type
```